src/runner/settings_window.cpp

```diff
@@ -250,6 +251,23 @@ void run_settings_window()
         settings_theme = L"dark";
     }
 
+    // Arg 4: settings theme.
+    GeneralSettings save_settings = get_general_settings();
+
+    bool isElevated{ get_general_settings().isElevated };
+    std::wstring settings_elevatedStatus;
```

Comment:
Is this string needed?

Reply:
yes, its is used in now In settings a start-up argument.